h2o-docs/src/product/data-science/algo-params/distribution.rst

```diff
@@ -80,6 +80,87 @@ Some examples of response distributions are provided below.
 .. figure:: ../../images/bernoulli.png
    :alt: Bernoulli distribution
 
+Equations
+~~~~~~~~~
+
+Where:
+
+- :math:`y` is a true response
+- :math:`f` is a predicted response
+- :math:`w` is weight
+
++-------------------+-------------------------------------------------------------------------------------------+
+| Distribution Type | Equation                                                                                  |
++===================+===========================================================================================+
+| Bernoulli         | :math:`-2w(y*log(f))+(1-y)*log(1-f))`                                                     |
++-------------------+-------------------------------------------------------------------------------------------+
+| Quasibinomial     | - :math:`y==f:0`                                                                          |
+|                   | - :math:`f>1: -2w*y*log(f)`                                                               |
+|                   | - :math:`f<0: -2w*(1-y)*log(1-f) \text{ else}`                                            |
+|                   | - :math:`-2w*(y*log(f)+(1-y)*log(1-f))`                                                   |
++-------------------+-------------------------------------------------------------------------------------------+
+| Gaussian          | - Equivalent to wMSE (weighted mean squared error):                                       |
+|                   | - :math:`w*(y-f)^2`                                                                       |
++-------------------+-------------------------------------------------------------------------------------------+
+| Poisson           | - With :math:`f=link(f)`                                                                  |
+|                   | - :math:`-2w*(y*f-exp(f))`                                                                |
++-------------------+-------------------------------------------------------------------------------------------+
+| Gamma             | - With :math:`f=link(f)`                                                                  |
+|                   | - :math:`2w*(y*exp(-f)+f)`                                                                |
++-------------------+-------------------------------------------------------------------------------------------+
+| Laplace           | :math:`w*\mid{y-f}\mid`                                                                   |
++-------------------+-------------------------------------------------------------------------------------------+
+| Quantile          | - :math:`y>f: w*QuantileAlpha*(y-f)`                                                      |
+|                   | - :math:`y<=f: w*1-QuantileAlpha*(f-y)`                                                   |
```

Comment:
Should have parentheses around 1-QuantilAlpha
`w*(1-QuantileAlpha)*(f-y)`
From https://github.com/h2oai/h2o-3/blob/c395633dd20346bded1da3f87cc17f4d6b977a84/h2o-core/src/main/java/hex/DistributionFactory.java#L430